data/scripts/quests/#example.lua

```diff
@@ -0,0 +1,21 @@
+local quest = Game.createQuest("Example Quest II")
+quest.storageId = 1002
+quest.storageValue = 1
```

Comment:
What do you think of using tables for parameters?

```
Game.createQuest("Example Quest II", {
  storageId = 1002,
  storageValue = 1,

  missions = {{
    name = "Example Mission 1",
    ...
  }, {
    ...
  }}
})
```

Then you don't need to handle index/newindex